pytradfri/const.py

```diff
@@ -109,14 +109,25 @@
 # URL to json-file containing links to all firmware updates
 URL_OTA_FW = "http://fw.ota.homesmart.ikea.net/feed/version_info.json"
 
+
 # Mireds range for which the conversion functions work
 # and that RGB bulbs can show
-MIN_MIREDS = 40
-MAX_MIREDS = 600
-
+RANGE_MIREDS = (40, 600)
```

Comment:
Has anyone successfully set the color temp of a color bulb? The features parameter of color bulbs tells us that you can't and trying it also doesn't do anything.

But at the same time this comment references the ability to set the color temp of a color bulb.

Since I can't set the color temp of a color bulb I'd like to simplify this and only use `RANGE_MIREDS_WS` as `RANGE_MIREDS`. But if someone managed to do this I'd very much like to know how they did it.

Reply:
My recollection is that only hex and x/y could be used to change colors. In some cases, the colors were changed on the gateway but nothing happened with the bulbs itself. But it's been a few weeks since I played around with color so I'm 100% sure.

Reply:
You can also use HSB, but apparently not Mireds. Very annoying, but it is what it is. I'll simplify it like described above.